packages/client/webapp/src/App.vue

```diff
@@ -1,19 +1,23 @@
 <script setup lang="ts">
-import { onBeforeMount, computed } from 'vue';
+import { onBeforeMount, ref, computed } from 'vue';
 import Header from '@/components/Header.vue';
+import Sidebar from '@/components/Sidebar.vue';
 import { useAuthStore } from './stores/auth';
 
 const auth = useAuthStore();
-
 /**
  * Before mounting go fetch the SSO
  * to make sure user has credentials to view app
  */
 onBeforeMount(() => {
 	auth.fetchSSO();
+	if (localStorage.sidebarPosition === undefined) {
+		localStorage.setItem('sidebarPosition', 'left');
+	}
 });
 
 const isAuthenticated = computed(() => auth.isAuthenticated);
+const sidebarPosition = ref(localStorage.sidebarPosition);
```

Comment:
Nitpick: You may wish to get used to using the syntax:
```suggestion
const sidebarPosition = ref(localStorage.getItem('sidebarPosition'));
```

The only differences are:
- `getItem` returns `null` instead of `undefined` when the item is missing, and
- `getItem` will behave consistently for any item name, whereas the `localStorage.x` syntax will be unpredictable if `x` is any of the 6 existing properties of localStorage, most notably `length` or `key`.

See [this stackoverflow response (specifically the EDIT)](https://stackoverflow.com/a/15439059) for a more detailed answer.